### ISSUE TITLE
Issues with Canvas setting

### ISSUE THREAD

**Issue by Appster-Arun** [kmbang/UnityNativeEdit#18]
First of all thanks for contributing this super awesome utility.
It does not work, when you change the canvas render mode from Screen Space-Overlay to ScreenSpace-Camera. 
Although I am doing my research, since you guys know the plugin best please point me what can we do to resolve this issue.

Cheers,
Arun

**Comment by anamtakhan93** [comment]
Yeah I just tried this and it doesn't work on screen space camera.
plus I can't touch anything else on unity engine UI :/

**Comment by si-bada** [comment]
even with overplay does not work ... well it works but i can't click on anything UI reltaed object :((((((
